apps/daimo-mobile/src/logic/notify.ts

```diff
@@ -62,6 +65,30 @@ class PushNotificationManager {
 
   maybeSavePushTokenForAccount = async () => {
     try {
+      if (Platform.OS === "android") {
+        // Android specific setup
+        await Notifications.setNotificationChannelAsync('default', {
+          name: 'default',
+          importance: Notifications.AndroidImportance.MAX,
+          vibrationPattern: [0, 250, 250, 250],
+          lightColor: '#FF231F7C',
```

Comment:
do `lightColor`/`vibrationPattern` have reasonable defaults?

Reply:
@dcposch Yeah, it's comfortable I would say